### DISCUSSION TITLE
System Path Entry Where??

### DISCUSSION THREAD

**Discussion by NullPlane** [microsoft/winget-cli#4325] [Q&A] [answered]
Where does winget register apps without an official installer that are meant to be used through console?
To be more precise, I installed [`schollz/croc`](https://github.com/schollz/croc "Easily and securely send things from one computer to another 🐊 📦. schollz.com/software/croc6"), however it does not appear neither in System or User PATH
nor is it to be found in `HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\` and yet, it is 
still perfectly usable. So how is it registered and where?

**Comment by Trenly** [comment, marked as answer]
If portable applications are installed with Administrator privileges, or if Developer Mode is enabled on the machine, then the `Portable Links Directory` is added to `%PATH%`. If not installed using Administrator or Developer mode, then the full path to the application directory is added to `%PATH%`. 

You can see the `Portable Links Directory` path in the output of `winget --info`.
